asciidoctorj-core/build.gradle

```diff
@@ -32,3 +34,19 @@ jrubyPrepareGems << {
 //jruby {
 //	execVersion = '1.7.20'
 //}
+
+jar {
+  manifest {
+    symbolicName = 'org.asciidoctor'
+    instruction 'Export-Package',
+      "org.asciidoctor;version=\"${version}\"",
+      "org.asciidoctor.ast;version=\"${version}\"",
+      "org.asciidoctor.converter;version=\"${version}\"",
+      "org.asciidoctor.converter.spi;version=\"${version}\"",
+      "org.asciidoctor.extension;version=\"${version}\"",
+      "org.asciidoctor.extension.spi;version=\"${version}\""
+    instruction 'Import-Package',
+      'com.beust.jcommander;resolution:=optional',
```

Comment:
Absolutely. It means that this import isn't required, so that the bundle will resolve even if the package `com.beust.jcommander` isn't available. All other packages are required though.

The gradle osgi plugin uses the bnd tool (https://github.com/bndtools/bnd) which processes these lines to produce the actual JAR and the final MANIFEST.MF.

You can also see that `org.asciidoctor.cli` is not part of the exported packages. It is still in the JAR but not exported, thus in an OSGi environment, other bundles can not use/see it.